specifyweb/frontend/js_src/lib/components/Formatters/Formatter.tsx

```diff
@@ -224,24 +213,26 @@ function ConditionalFormatter({
     field: Formatter['definition']['fields'][number],
     index: number
   ) => void;
-  readonly showConditionalField: GetOrSet<WritableArray<boolean>>;
   readonly trimmedFieldsLength: number;
   readonly formatter: GetSet<Formatter>[0];
   readonly onChange: (fields: Formatter['definition']['fields']) => void;
   readonly table: SpecifyTable;
-  readonly onRemoveField: (index: number) => void;
 }): JSX.Element {
   const isReadOnly = React.useContext(ReadOnlyContext);
+  const [collapsedConditionalField, _, __, toggleConditionalField] =
```

Comment:
since you are inside a component called `ConditionalFormatter` there is no need for a name as verbose as `collapsedConditionalField`

`isCollapsed` would work great here!

a verbose name like `collapsedConditionalField` is needed if it was global variable that you export and could import from any other file - in that case the verbosity is required to give context of what the variable is about, and to reduce chance of collision with other exported variable names